gravitee-apim-console-webui/src/management/application/application-navigation/application-navigation.component.ts

```diff
@@ -129,6 +122,10 @@ export class ApplicationNavigationComponent implements OnInit, OnDestroy {
     ]);
 
     this.selectedItemWithTabs = this.subMenuItems.find((item) => item.tabs && this.isTabActive(item.tabs));
```

Comment:
so you can remove this line as now you use the startWith to init the observable below

Reply:
Done